lang/en/validation.php

```diff
@@ -133,7 +133,7 @@
     /*
      * Strict validation rules
      */
-
+    'contains_regular_character' => 'The :attribute must contain at least one regular character.',
```

Comment:
I can't seem to trigger this. Everything I paste returns "Body is required"

Reply:
Try hex C2AD (search message_threads for id = 159237 and copy the `title` value).

![Screenshot 2024-09-07 at 6 14 04 PM](https://github.com/user-attachments/assets/9060b949-f3fe-483a-be1a-f7819b098e62)